src/test/java/com/alipay/remoting/rpc/exception/ExceptionTest.java

```diff
@@ -341,4 +341,29 @@ public Executor getExecutor() {
         latch.await();
         Assert.assertEquals(InvokeServerException.class, ret.get(0).getClass());
     }
+
+    @Test
+    public void testGetBizClassLoaderException1() {
+        server.registerUserProcessor(new SimpleServerUserProcessor() {
+            @Override
+            public ClassLoader getBizClassLoader() {
+                throw new RuntimeException("getBizClassLoader fail.");
+            }
+        });
+
+        RequestBody req = new RequestBody(4, "hello world");
+        try {
+            client.invokeSync(addr, req, 3000);
+        } catch (InvokeServerException e) {
+            Assert.assertTrue(true);
```

Comment:
This statement is meaningless.

Reply:
This statement indicate that  must throw InvokeServerException.

Please refer to method : `ExceptionTest#testSyncException`

[ExceptionTest.java#L187](https://github.com/JoeCqupt/sofa-bolt/blob/master/src/test/java/com/alipay/remoting/rpc/exception/ExceptionTest.java#L187)

Reply:
`Assert.assertTrue(true);` always true and do nothing ,weather or not this statement is run, takes no difference.

Reply:
done